src/SOS/SOS.UnitTests/SOSRunner.cs

```diff
@@ -1216,6 +1225,10 @@ private HashSet<string> GetEnabledDefines()
         {
             defines.Add("NETCORE_OR_DOTNETDUMP");
         }
+        if (_config.PublishSingleFile)
+        {
+            defines.Add("SINGLE_FILE_APP");
```

Comment:
Do we use this anywhere? Is this for the test scripts to be able to check if it's a single file?